ddtrace/settings/config.py

```diff
@@ -229,7 +229,7 @@ def __init__(self):
 
         self._telemetry_enabled = asbool(os.getenv("DD_INSTRUMENTATION_TELEMETRY_ENABLED", True))
 
-        self._telemetry_metrics_enabled = asbool(os.getenv("_DD_TELEMETRY_METRICS_ENABLED", default=False))
+        self._telemetry_metrics_enabled = asbool(os.getenv("DD_TELEMETRY_METRICS_ENABLED", default=True))
```

Comment:
DD_TELEMETRY_METRICS_ENABLED  is not an officially supported environment variable. I think we should keep it private and avoid adding it to our public docs.

https://github.com/DataDog/instrumentation-telemetry-api-docs/blob/main/GeneratedDocumentation/ApiDocs/v2/how-to-use.md#telemetry-environment-variables